src/api/tasks/TaskRouter.router.js

```diff
@@ -0,0 +1,82 @@
+"use strict";
+
+const { Router } = require("express");
+const { fail } = require("../../utils/express/response");
+
+const {
+  sendRegistrationToTascomiAction,
+  sendNotificationsForRegistrationAction,
+  saveRegistrationToTempStoreAction,
+  sendAllOutstandingRegistrationsToTascomiAction,
+  sendAllNotificationsForRegistrationsAction,
+  saveAllOutstandingRegistrationsToTempStoreAction
+} = require("./Tasks.controller");
+
+// const { viewDeleteRegistrationAuth } = require("../../middleware/authHandler");
+
+const TaskRouter = () => {
+  const router = Router();
+
+  //apply any middleware
+  //router.use(viewDeleteRegistrationAuth);
+
+  router.get("/bulk/createtascomiregistration", async (req, res) => {
+    let dryrun = !!req.query.dryrun;
+    try {
+      await sendAllOutstandingRegistrationsToTascomiAction(req, res, dryrun);
+    } catch (e) {
+      await fail(406, res, e.message);
```

Comment:
are these not generic 500s?

Reply:
As per chat - we will let slide and revise later.